server/utils/setupEventWebhook.js

```diff
@@ -35,7 +35,9 @@ const setupEventWebhook = (strapi, settings) => {
 
 	for (const [event, eventModels] of Object.entries(events)) {
 		strapi.eventHub.on(event, (data) => {
-			if (eventModels.includes(data.model)) {
+			// build on data.media (nothing to verify since it wouldn't get here without having the event setup on the event listener)
+			// otherwise, build on matching models
+			if (!!data.media || eventModels.includes(data.model)) {
```

Comment:
What is the reason behind using `!!data.media` over `data.media`?

Reply:
I've run into issues in the past using Typescript looking specifically for booleans and so I guess I just got in a habit of defaulting to the boolean casting. I will update it.

Reply:
Also, in regards to your comment...
> Its strange as media does have a model (files), not sure why it is not included.

Perhaps the file model worked for all media events (create, update, delete) in the past. But, while doing my testing, it looks like the file model only fires when the media file is updated. It doesn't show up for a create or delete event. So, I thought it was safer to look for `data.media` rather than `data.model === 'file'`. That is also the reason why I ignore file models in my other PR regarding the wildcard event trigger (#25)